posthog/cdp/templates/intercom/template_intercom.py

```diff
@@ -1,135 +1,296 @@
-from copy import deepcopy
-import dataclasses
-from posthog.cdp.templates.hog_function_template import HogFunctionTemplate, HogFunctionTemplateMigrator
-
+from posthog.cdp.templates.hog_function_template import HogFunctionTemplate
 
 template: HogFunctionTemplate = HogFunctionTemplate(
     status="beta",
     type="destination",
-    id="template-Intercom",
+    id="template-intercom",
```

Comment:
If you are changing the ID, ideally we would migrate all existing hog_functions so the template link stays the same please.